src/opart_gaussian.c

```diff
@@ -2,13 +2,70 @@
 #include "opart_gaussian.h"
 
 
```

Comment:
Hi @tdhock , I removed the function declarations as discussed. I will update the variable names in next push.